docs/_plugins/copy_api_dirs.rb

```diff
@@ -150,4 +150,31 @@
     cp("../R/pkg/DESCRIPTION", "api")
   end
 
+  if not (ENV['SKIP_SQLDOC'] == '1')
+    # Build SQL API docs
+
+    puts "Moving to project root and building API docs."
+    curr_dir = pwd
+    cd("..")
```

Comment:
I think I misunderstood your previous comments initially. It shouldn't be hard.

but other language API docs are, up to my knowledge, in separate directories and then copied into `docs/` later. I was thinking we could extend this further in the future (e.g., syntax documentation) and it could be easier to check doc output in a separate dir (actually, for me, I check other docs output in this way more often).